ios/ci_scripts/ci_pre_xcodebuild.sh

```
@@ -0,0 +1,49 @@
+# !/bin/bash
```

Shellcheck

[error] Remove spaces between # and ! in the shebang.

(SC1115)